pulp_smash/tests/platform/cli/test_content_sources.py

```diff
@@ -0,0 +1,97 @@
+# coding=utf-8
+"""Tests for list/refresh/delete content sources."""
+from __future__ import unicode_literals
+
+import unittest2
+from packaging.version import Version
+
+from pulp_smash import cli, config
+from pulp_smash.constants import CONTENT_SOURCE_ID
+
+
+def _get_content_source_ids(server_config):
+    """Get the id list of all content sources, or None if the list is empty.
+
+    :param server_config: pulp_smash.config.ServerConfig server_config:
+        Information about the Pulp server being targeted.
+    :return: None if the list of content sources is empty, or the IDs of all
+        content sources as ``list``.
+    """
+    keyword = 'Source Id:'
+    completed_proc = cli.Client(server_config).run(
+        'pulp-admin content sources list'.split()
+    )
+    lines = [
+        line for line in completed_proc.stdout.splitlines()
+        if keyword in line
+    ]
+    assert len(lines) >= 0
+    if len(lines) == 0:
+        return None
```

Comment:
Imagine the Pulp server has content sources with IDs 15, 323 and 11. In this case, this function returns `[15, 323, 11]`.

Imagine the Pulp server has content sources with IDs 15 and 323. In this case, this function returns `[15, 323]`.

Imagine the Pulp server has a content source with an ID of 15. In this case, the function returns `[15]`.

Imagine the Pulp server has no content sources. Shouldn't the function return `[]`?